lib/root/widgets/cart_icon.dart

```diff
@@ -13,6 +13,10 @@ class CartIcon extends StatelessWidget {
     final count =
         context.select((CartIconCubit cubit) => cubit.state.totalItems);
 
+    if (count == 0) {
+      return const Icon(LineAwesome.shopping_bag_solid);
```

Comment:
We should DRY this, in case we change the icon 😇 

```dart
var widget = const Icon(LineAwesome.shopping_bag_solid);

if (count > 0) {
  result = Badge(
    ...
    ...
    ...
    child: result;
  );
}
```

or, if you dislike this pattern with the reassign:

```dart
  var widget = const Icon(LineAwesome.shopping_bag_solid);

  if (count == 0) {
    return widget;
  }

  return Badge(
    ...
    ...
    ...
    child: result;
  );
```